File: IDExamination/IDSeatingArrangement/IDSeatingArrangement/Seating.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace IDSeatingArrangement
{
    class Seating
    {
        public string Id { get; set; }
        public string Course { get; set; }
        public string T1Date { get; set; }
        public string T2Date { get; set; }
        public string Time { get; set; }
        public string Room { get; set; }
        public string Students { get; set; }
        public Boolean Compre { get; set; }
        public Seating(string c1, string t1, string t2, string time, string room, string students,Boolean compre )
        {
            Course = c1;T1Date = t1;T2Date = t2;Time = time;Room = room;Students = students;Compre = compre;
        }
    }
}

File: Android/BPHC-ID/app/src/main/java/com/srujan/bphc_id/Seating.java
package com.srujan.bphc_id;

/**
 * Created by Srujan on 10/14/2015.
 */
public class Seating {
    public String Id;
    public String course;
    public String t1date;
    public String t2date;
    public String time;
    public String room;
    public String students;
    public Boolean compre;
    public Seating(String c1, String t1, String t2, String t, String r, String stus,Boolean co ) {
        course = c1;
        t1date = t1;
        t2date = t2;
        time = t;
        room = r;
        students = stus;
        compre = co;
    }
}
